src/granite/table.cr

```diff
@@ -1,8 +1,18 @@
+# Adds a :nodoc: to granite methods/constants if `DISABLE_GRANTE_DOCS` ENV var is true
+macro noDoc(stmt)
+  {% if env("DISABLE_GRANTE_DOCS") == "true" %}
```

Comment:
The purpose here is to make the docs more terse and relevant by default, right? Should this logic default to nodoc then?

Reply:
Depends on how you look at it.  Was trying to allow you to disable the docs if you wanted while retaining current behavior.